libs/wingsdk/test/target-tf-aws/api.test.ts

```diff
@@ -0,0 +1,16 @@
+import { Api } from "../../src/cloud";
+import * as tfaws from "../../src/target-tf-aws";
+import { mkdtemp } from "../../src/util";
+import { tfResourcesOf, tfSanitize, treeJsonOf } from "../util";
+
+test("api with GET route", () => {
```

Comment:
We should probably include these as test cases:
- api with no routes
- api with one GET route
- api with two different GET routes
- api with two different GET routes, where one is a prefix of the other (e.g. "/hello" and "/hello/world")
- throws when get() is called with the same route twice

Reply:
WIP I added some test cases :smile: